### PR TITLE
Add Implementation guide library module

### PR DESCRIPTION
**Pull request by ktarasenko**
**Description**
Introduce an empty module for Implementation guide management library 

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://github.com/google/android-fhir/wiki/Contributing) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import androidx.build.gradle.gcpbuildcache.GcpBuildCache
 import androidx.build.gradle.gcpbuildcache.GcpBuildCacheServiceFactory
 
+include()
```
Comment (jingtang10) [inline settings.gradle.kts:4]: are these needed?

**Review by jingtang10 — CHANGES_REQUESTED** [review]

File: settings.gradle.kts (hunk 1 above)
Comment (ktarasenko) [reply settings.gradle.kts]: removed

File: implementationguide/proguard-rules.pro [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#   public *;
+#}
+
+# Uncomment this to preserve the line number information for
+# debugging stack traces.
+#-keepattributes SourceFile,LineNumberTable
+
+# If you keep the line number information, uncomment this to
+# hide the original source file name.
+#-renamesourcefileattribute SourceFile
```
[the remark this answers is not in the document]
Comment (ktarasenko) [reply implementationguide/proguard-rules.pro]: done

File: buildSrc/src/main/kotlin/Releases.kt [hunk 3]
```diff
@@ -72,6 +72,12 @@ object Releases {
     }
   }
 
+  object ImplmentationGuide : LibraryArtifact {
+    override val artifactId = "implementationguide"
+    override val version = "0.1.0-alpha001"
+    override val name = "Android FHIR Implementation Guide management Library"
```
Comment (jingtang10) [inline buildSrc/src/main/kotlin/Releases.kt:78]:
```suggestion
    override val name = "Android FHIR Implementation Guide Library"
```

**Review by jingtang10 — APPROVED** [review]

**Pull request merged**